src/output.zig

```diff
@@ -805,9 +805,9 @@ fn ScopedLogger(comptime tagname: []const u8, comptime visibility: Visibility) t
             if (bun.getenvZAnyCase("BUN_DEBUG_" ++ tagname)) |val| {
                 really_disable.store(strings.eqlComptime(val, "0"), .monotonic);
             } else if (bun.env_var.BUN_DEBUG_ALL.get()) |val| {
-                really_disable.store(val, .monotonic);
+                really_disable.store(!val, .monotonic);
             } else if (bun.env_var.BUN_DEBUG_QUIET_LOGS.get()) |val| {
-                really_disable.store(really_disable.load(.monotonic) or !val, .monotonic);
+                really_disable.store(really_disable.load(.monotonic) or val, .monotonic);
```

Comment:
This is unrelated but

```suggestion
                _ = really_disable.fetch_or(val, .monotonic);
```